packages/ibm-products/src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -28,44 +28,97 @@ const componentName = 'UserAvatar';
 
 // NOTE: the component SCSS is not imported here: it is rolled up separately.
 
-// Default values can be included here and then assigned to the prop params,
-// e.g. prop = defaults.prop,
-// This gathers default values together neatly and ensures non-primitive
-// values are initialized early to avoid react making unnecessary re-renders.
-// Note that default values are not required for props that are 'required',
-// nor for props where the component can apply undefined values reasonably.
-// Default values should be provided when the component needs to make a choice
-// or assumption when a prop is not supplied.
-
-// Default values for props
-// const defaults = {
-//   /* TODO: add defaults for relevant props if needed */
-// };
-
 /**
  * TODO: A description of the component.
  */
+type BackgroundColor =
+  | 'order-1-cyan'
+  | 'order-2-gray'
+  | 'order-3-green'
+  | 'order-4-magenta'
+  | 'order-5-purple'
+  | 'order-6-teal'
+  | 'order-7-cyan'
+  | 'order-8-gray'
+  | 'order-9-green'
+  | 'order-10-magenta'
+  | 'order-11-purple'
+  | 'order-12-teal';
+type Size = 'xl' | 'lg' | 'md' | 'sm';
+type TooltipAlignment =
+  | 'top'
+  | 'top-left'
+  | 'top-right'
+  | 'bottom'
+  | 'bottom-left'
+  | 'bottom-right'
+  | 'left'
+  | 'right';
+
+type ImageProps =
+  | {
+      /**
+       * When passing the image prop, supply a full path to the image to be displayed.
+       */
+      image: string;
+      /**
+       * When passing the image prop use the imageDescription prop to describe the image for screen reader.
+       */
+      imageDescription: string;
+    }
+  | {
+      image?: never;
+      imageDescription?: never;
+    };
```

Comment:
Hey @anamikaanu96 , I'm not sure why we need the never types optionally in the second object?